content/judgment.tex

```diff
@@ -40,9 +40,19 @@ \subsection{Operator of the GameController}
 The operator will also inform the assistant referees when a timed penalty is over and a robot has to be placed back on the field.
 They should announce when the ball is in play on kick-off by stating ``Ball Free'', if the \KickOffBallFreeTime time period has elapsed in the playing state.
 They are also responsible for keeping the time of each half (\ie, they stop the clock after a goal or game stuck, and continues it at the kick-off\footnote{The clock may not be stopped during the preliminaries.}).
-They should count aloud the remaining seconds in a half once the time remaining is 5 seconds or less.
+They should count aloud the remaining seconds in a half once the time remaining is \qty{5}{\second} or less.
 Finally, they should repeat the calls of the head referee to make sure it was heard correctly.
 
+\subsection{Referee — Team communication}
+\label{sec:referee_team_communiation}
+Both teams send a representative called team captain to the field \qty{10}{\minute} before their match starts. This time should be used for welcome each other, discuss match related topics.
+
+During a match only the team captains are communicating with the head referee. Only the team captains and two more people per teams are allowed to stay next to the game controller tables. The rest of the team keeps distance. This allows the referees easier communication with the team and the game controller operator gets less disturbed.
```

Comment:
The rest of the team keeps distance **around the other sides of the field.**

Reply:
Thanks, got addressed